addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -129,6 +129,8 @@ function factory(dependencies) {
                         partner_id,
                         partner_name,
                     });
+                case 'channel_updated':
+                    return this._handleNotificationChannelUpdated(channelId, data);
```

Comment:
We need to stop looking on the "bus channel" to determine target of payload. The handler should only receive the data, and it should contain the actual channel id inside it.